proto/prysm/v1alpha1/attestation.go

```diff
@@ -140,6 +153,14 @@ func (a *PendingAttestation) Version() int {
 	return version.Phase0
 }
 
+// IsNil --
+func (a *PendingAttestation) IsNil() bool {
+	if a == nil || a.Data == nil {
+		return true
+	}
+	return false
+}
+
```

Comment:
Could you not just return the value?
 Same for the others too
 ```suggestion
// IsNil --
func (a *PendingAttestation) IsNil() bool {
	return a == nil || a.Data == nil
}
```

Reply:
Good idea will do that